README.md

```diff
@@ -24,8 +24,8 @@ import (
 )
 
 func main() {
-    serviceLoader := authservice.NewRemoteServiceRegLoader("https://rokwire.illinois.edu/auth", nil)
-	authService, err := authservice.NewAuthService("example", "https://rokwire.illinois.edu/example", serviceLoader)
+    dataLoader := authservice.NewRemoteAuthDataLoader("https://rokwire.illinois.edu/auth", "/access-token", "example_token", "/service-regs", nil)
```

Comment:
I think maybe we can provide default values for these paths and make them optional parameters. To do this, we could define a config struct that is passed to this function as an optional parameter.